.harness/publish_to_docker.yaml

```diff
@@ -179,3 +179,4 @@ pipeline:
       required: true
       value: <+input>
   allowStageExecutions: true
+  description: please add description
```

Comment:
@NolanJMcCafferty 
Could you please add pipeline description.
I suggest to specify: what pipeline does, when it is triggered.